app/blend.cc

```diff
@@ -0,0 +1,63 @@
+#include "utils.h"
+#include <opencv2/imgproc/imgproc.hpp>
+
+cv::Mat alpha_blend(const cv::Mat& srca, const cv::Mat& srcb, const cv::Mat& mask) {
+	// alpha blend two (8UC3) source images using a mask (8UC1, 255=>srca, 0=>srcb), adapted from:
+	// https://www.learnopencv.com/alpha-blending-using-opencv-cpp-python/
+	// "trust no-one" => we're about to mess with data pointers
+	assert(srca.rows == srcb.rows);
+	assert(srca.cols == srcb.cols);
+	assert(mask.rows == srca.rows);
+	assert(mask.cols == srca.cols);
+
+	assert(srca.type() == CV_8UC3);
+	assert(srcb.type() == CV_8UC3);
+	assert(mask.type() == CV_8UC1);
+
+	const uint8_t *aptr = (const uint8_t*)srca.data;
+	const uint8_t *bptr = (const uint8_t*)srcb.data;
+	const uint8_t *mptr = (const uint8_t*)mask.data;
+
+	cv::Mat out = cv::Mat::zeros(srca.size(), srca.type());
```

Comment:
As we're writing to all elements later anyway, is there a way to just set dimensions and type, without "initializing" this? No need to change this, just an idea …

Reply:
Can likely switch to the `Mat::create` method to avoid writing to elements.

Reply:
Nice. Reduced timings to 0.6-1.8 msecs :smile: